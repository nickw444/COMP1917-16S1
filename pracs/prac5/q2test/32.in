32
-7 -6 3 2 0 -8 -6 -1 -2 10 5 -6 -3 5 3 -1 -7 3 9 7 2 9 -10 2 2 -6 -10 -10 8 -6 0 -8
