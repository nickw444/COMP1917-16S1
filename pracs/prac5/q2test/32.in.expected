average of array: -0.843750
